### PR TITLE
static systemd unit : fix idempotent issue

### LINKED ISSUE #46247: service: disabling twice is not idempotent
##### SUMMARY

Disable a service twice. The 2nd time it returns "changed".

##### ISSUE TYPE
- Bug Report

##### COMPONENT NAME

service

##### ANSIBLE VERSION

```paste below
ansible 2.6.4
  config file = None
  configured module search path = ['/root/.ansible/plugins/modules', '/usr/share/ansible/plugins/modules']
  ansible python module location = /usr/local/lib/python3.6/dist-packages/ansible
  executable location = /usr/local/bin/ansible
  python version = 3.6.6 (default, Sep 12 2018, 18:26:19) [GCC 8.0.1 20180414 (experimental) [trunk revision 259383]]
```

##### CONFIGURATION

N/A

##### OS / ENVIRONMENT

Several.

##### STEPS TO REPRODUCE

From https://github.com/ansible/ansible/issues/31344#issuecomment-419412368:

```bash
docker container exec -it $(docker container run -d -v /sys/fs/cgroup:/sys/fs/cgroup:ro --tmpfs /tmp --tmpfs /run --tmpfs /run/lock tecnativa/ansible-test-playground:ubuntu-18.04) sh -c 'apt update && apt install -yqq python3-pip && pip3 install ansible==2.6.4 && chmod +x /usr/local/bin/ansible* && ansible --version && ansible localhost -m service -a "name=systemd-journald state=stopped enabled=false" -e ansible_connection=local -vvvv && ansible localhost -m service -a "name=systemd-journald state=stopped enabled=false" -e ansible_connection=local' -vvvv
```

:point_up: There you're running a systemd-enabled ubuntu 18.04 container, installing Ansible on python 3, and disabling the `systemd-journald` service twice. 

Remember to stop the container when you finish.

##### EXPECTED RESULTS

1st time it should be changed, as it is. The 2nd time it is expected to return `ok`, but it returns `changed`.

##### ACTUAL RESULTS

```paste below
No config file found; using defaults
setting up inventory plugins
 [WARNING]: Unable to parse /etc/ansible/hosts as an inventory source

 [WARNING]: No inventory was parsed, only implicit localhost is available

 [WARNING]: provided hosts list is empty, only localhost is available. Note
that the implicit localhost does not match 'all'

Loading callback plugin minimal of type stdout, v2.0 from /usr/local/lib/python3.6/dist-packages/ansible/plugins/callback/minimal.py
META: ran handlers
<127.0.0.1> ESTABLISH LOCAL CONNECTION FOR USER: root
<127.0.0.1> EXEC /bin/sh -c 'echo ~root && sleep 0'
<127.0.0.1> EXEC /bin/sh -c '( umask 77 && mkdir -p "` echo /root/.ansible/tmp/ansible-tmp-1538118794.9681993-230714754075598 `" && echo ansible-tmp-1538118794.9681993-230714754075598="` echo /root/.ansible/tmp/ansible-tmp-1538118794.9681993-230714754075598 `" ) && sleep 0'
Using module file /usr/local/lib/python3.6/dist-packages/ansible/modules/system/setup.py
<127.0.0.1> PUT /root/.ansible/tmp/ansible-local-4063oc6ihw27/tmpunfqo2al TO /root/.ansible/tmp/ansible-tmp-1538118794.9681993-230714754075598/setup.py
<127.0.0.1> EXEC /bin/sh -c 'chmod u+x /root/.ansible/tmp/ansible-tmp-1538118794.9681993-230714754075598/ /root/.ansible/tmp/ansible-tmp-1538118794.9681993-230714754075598/setup.py && sleep 0'
<127.0.0.1> EXEC /bin/sh -c '/usr/bin/python3 /root/.ansible/tmp/ansible-tmp-1538118794.9681993-230714754075598/setup.py && sleep 0'
Running systemd
Using module file /usr/local/lib/python3.6/dist-packages/ansible/modules/system/systemd.py
<127.0.0.1> PUT /root/.ansible/tmp/ansible-local-4063oc6ihw27/tmp4teg21sc TO /root/.ansible/tmp/ansible-tmp-1538118794.9681993-230714754075598/systemd.py
<127.0.0.1> EXEC /bin/sh -c 'chmod u+x /root/.ansible/tmp/ansible-tmp-1538118794.9681993-230714754075598/ /root/.ansible/tmp/ansible-tmp-1538118794.9681993-230714754075598/systemd.py && sleep 0'
<127.0.0.1> EXEC /bin/sh -c '/usr/bin/python3 /root/.ansible/tmp/ansible-tmp-1538118794.9681993-230714754075598/systemd.py && sleep 0'
<127.0.0.1> EXEC /bin/sh -c 'rm -f -r /root/.ansible/tmp/ansible-tmp-1538118794.9681993-230714754075598/ > /dev/null 2>&1 && sleep 0'
localhost | SUCCESS => {
    "changed": true,
    "enabled": false,
    "invocation": {
        "module_args": {
            "daemon_reload": false,
            "enabled": false,
            "force": null,
            "masked": null,
            "name": "systemd-journald",
            "no_block": false,
            "state": "stopped",
            "user": false
        }
    },
    "name": "systemd-journald",
    "state": "stopped",
    "status": {
        "ActiveEnterTimestamp": "Fri 2018-09-28 07:10:48 UTC",
        "ActiveEnterTimestampMonotonic": "475068620",
        "ActiveExitTimestampMonotonic": "0",
        "ActiveState": "active",
        "After": "systemd-journald.socket system.slice systemd-journald-dev-log.socket syslog.socket systemd-journald-audit.socket",
        "AllowIsolate": "no",
        "AmbientCapabilities": "",
        "AssertResult": "yes",
        "AssertTimestamp": "Fri 2018-09-28 07:10:48 UTC",
        "AssertTimestampMonotonic": "475047037",
        "Before": "sysinit.target",
        "BlockIOAccounting": "no",
        "BlockIOWeight": "[not set]",
        "CPUAccounting": "no",
        "CPUQuotaPerSecUSec": "infinity",
        "CPUSchedulingPolicy": "0",
        "CPUSchedulingPriority": "0",
        "CPUSchedulingResetOnFork": "no",
        "CPUShares": "[not set]",
        "CPUUsageNSec": "[not set]",
        "CPUWeight": "[not set]",
        "CacheDirectoryMode": "0755",
        "CanIsolate": "no",
        "CanReload": "no",
        "CanStart": "yes",
        "CanStop": "yes",
        "CapabilityBoundingSet": "cap_chown cap_dac_override cap_dac_read_search cap_fowner cap_setgid cap_setuid cap_sys_ptrace cap_sys_admin cap_audit_control cap_mac_override cap_syslog",
        "CollectMode": "inactive",
        "ConditionResult": "yes",
        "ConditionTimestamp": "Fri 2018-09-28 07:10:48 UTC",
        "ConditionTimestampMonotonic": "475047037",
        "ConfigurationDirectoryMode": "0755",
        "ControlGroup": "/system.slice/docker.service/system.slice/systemd-journald.service",
        "ControlPID": "0",
        "DefaultDependencies": "no",
        "Delegate": "no",
        "Description": "Journal Service",
        "DevicePolicy": "auto",
        "Documentation": "man:systemd-journald.service(8) man:journald.conf(5)",
        "DynamicUser": "no",
        "ExecMainCode": "0",
        "ExecMainExitTimestampMonotonic": "0",
        "ExecMainPID": "19",
        "ExecMainStartTimestamp": "Fri 2018-09-28 07:10:48 UTC",
        "ExecMainStartTimestampMonotonic": "475050155",
        "ExecMainStatus": "0",
        "ExecStart": "{ path=/lib/systemd/systemd-journald ; argv[]=/lib/systemd/systemd-journald ; ignore_errors=no ; start_time=[n/a] ; stop_time=[n/a] ; pid=0 ; code=(null) ; status=0/0 }",
        "FailureAction": "none",
        "FileDescriptorStoreMax": "4224",
        "FragmentPath": "/lib/systemd/system/systemd-journald.service",
        "GID": "[not set]",
        "GuessMainPID": "yes",
        "IOAccounting": "no",
        "IOSchedulingClass": "0",
        "IOSchedulingPriority": "0",
        "IOWeight": "[not set]",
        "IPAccounting": "no",
        "IPEgressBytes": "18446744073709551615",
        "IPEgressPackets": "18446744073709551615",
        "IPIngressBytes": "18446744073709551615",
        "IPIngressPackets": "18446744073709551615",
        "Id": "systemd-journald.service",
        "IgnoreOnIsolate": "no",
        "IgnoreSIGPIPE": "yes",
        "InactiveEnterTimestampMonotonic": "0",
        "InactiveExitTimestamp": "Fri 2018-09-28 07:10:48 UTC",
        "InactiveExitTimestampMonotonic": "475050226",
        "InvocationID": "7955be581bd0473dbc72c2c4d6a353eb",
        "JobRunningTimeoutUSec": "infinity",
        "JobTimeoutAction": "none",
        "JobTimeoutUSec": "infinity",
        "KeyringMode": "private",
        "KillMode": "control-group",
        "KillSignal": "15",
        "LimitAS": "infinity",
        "LimitASSoft": "infinity",
        "LimitCORE": "infinity",
        "LimitCORESoft": "infinity",
        "LimitCPU": "infinity",
        "LimitCPUSoft": "infinity",
        "LimitDATA": "infinity",
        "LimitDATASoft": "infinity",
        "LimitFSIZE": "infinity",
        "LimitFSIZESoft": "infinity",
        "LimitLOCKS": "infinity",
        "LimitLOCKSSoft": "infinity",
        "LimitMEMLOCK": "16777216",
        "LimitMEMLOCKSoft": "16777216",
        "LimitMSGQUEUE": "819200",
        "LimitMSGQUEUESoft": "819200",
        "LimitNICE": "0",
        "LimitNICESoft": "0",
        "LimitNOFILE": "16384",
        "LimitNOFILESoft": "16384",
        "LimitNPROC": "infinity",
        "LimitNPROCSoft": "infinity",
        "LimitRSS": "infinity",
        "LimitRSSSoft": "infinity",
        "LimitRTPRIO": "0",
        "LimitRTPRIOSoft": "0",
        "LimitRTTIME": "infinity",
        "LimitRTTIMESoft": "infinity",
        "LimitSIGPENDING": "59668",
        "LimitSIGPENDINGSoft": "59668",
        "LimitSTACK": "infinity",
        "LimitSTACKSoft": "8388608",
        "LoadState": "loaded",
        "LockPersonality": "yes",
        "LogLevelMax": "-1",
        "LogsDirectoryMode": "0755",
        "MainPID": "19",
        "MemoryAccounting": "no",
        "MemoryCurrent": "[not set]",
        "MemoryDenyWriteExecute": "yes",
        "MemoryHigh": "infinity",
        "MemoryLimit": "infinity",
        "MemoryLow": "0",
        "MemoryMax": "infinity",
        "MemorySwapMax": "infinity",
        "MountAPIVFS": "no",
        "MountFlags": "",
        "NFileDescriptorStore": "1",
        "NRestarts": "0",
        "Names": "systemd-journald.service",
        "NeedDaemonReload": "no",
        "Nice": "-1",
        "NoNewPrivileges": "no",
        "NonBlocking": "no",
        "NotifyAccess": "main",
        "OOMScoreAdjust": "0",
        "OnFailureJobMode": "replace",
        "PermissionsStartOnly": "no",
        "Perpetual": "no",
        "PrivateDevices": "no",
        "PrivateNetwork": "no",
        "PrivateTmp": "no",
        "PrivateUsers": "no",
        "ProtectControlGroups": "no",
        "ProtectHome": "no",
        "ProtectKernelModules": "no",
        "ProtectKernelTunables": "no",
        "ProtectSystem": "no",
        "RefuseManualStart": "no",
        "RefuseManualStop": "no",
        "RemainAfterExit": "no",
        "RemoveIPC": "no",
        "Requires": "systemd-journald.socket system.slice",
        "Restart": "always",
        "RestartUSec": "0",
        "RestrictNamespaces": "yes",
        "RestrictRealtime": "yes",
        "Result": "success",
        "RootDirectoryStartOnly": "no",
        "RuntimeDirectoryMode": "0755",
        "RuntimeDirectoryPreserve": "no",
        "RuntimeMaxUSec": "infinity",
        "SameProcessGroup": "no",
        "SecureBits": "0",
        "SendSIGHUP": "no",
        "SendSIGKILL": "yes",
        "Slice": "system.slice",
        "StandardError": "inherit",
        "StandardInput": "null",
        "StandardInputData": "",
        "StandardOutput": "null",
        "StartLimitAction": "none",
        "StartLimitBurst": "5",
        "StartLimitIntervalUSec": "10s",
        "StartupBlockIOWeight": "[not set]",
        "StartupCPUShares": "[not set]",
        "StartupCPUWeight": "[not set]",
        "StartupIOWeight": "[not set]",
        "StateChangeTimestamp": "Fri 2018-09-28 07:10:48 UTC",
        "StateChangeTimestampMonotonic": "475068620",
        "StateDirectoryMode": "0755",
        "StatusErrno": "0",
        "StatusText": "Processing requests...",
        "StopWhenUnneeded": "no",
        "SubState": "running",
        "SuccessAction": "none",
        "SyslogFacility": "3",
        "SyslogLevel": "6",
        "SyslogLevelPrefix": "yes",
        "SyslogPriority": "30",
        "SystemCallArchitectures": "native",
        "SystemCallErrorNumber": "0",
        "SystemCallFilter": "~_sysctl add_key adjtimex afs_syscall bdflush break chroot clock_adjtime clock_settime create_module delete_module finit_module ftime get_kernel_syms getpmsg gtty idle init_module ioperm iopl kexec_file_load kexec_load keyctl lock lookup_dcookie modify_ldt mount mpx pciconfig_iobase pciconfig_read pciconfig_write perf_event_open pivot_root process_vm_readv process_vm_writev prof profil ptrace putpmsg query_module reboot request_key rtas s390_pci_mmio_read s390_pci_mmio_write s390_runtime_instr security settimeofday sgetmask spu_run ssetmask stime stty swapoff swapon switch_endian sysfs tuxcall ulimit umount umount2 uselib ustat vm86 vm86old vserver",
        "TTYReset": "no",
        "TTYVHangup": "no",
        "TTYVTDisallocate": "no",
        "TasksAccounting": "yes",
        "TasksCurrent": "3",
        "TasksMax": "4915",
        "TimeoutStartUSec": "1min 30s",
        "TimeoutStopUSec": "1min 30s",
        "TimerSlackNSec": "50000",
        "Transient": "no",
        "TriggeredBy": "systemd-journald.socket systemd-journald-dev-log.socket systemd-journald-audit.socket",
        "Type": "notify",
        "UID": "[not set]",
        "UMask": "0022",
        "UnitFilePreset": "enabled",
        "UnitFileState": "static",
        "UtmpMode": "init",
        "WantedBy": "sysinit.target",
        "Wants": "systemd-journald.socket systemd-journald-dev-log.socket systemd-journald-audit.socket",
        "WatchdogTimestamp": "Fri 2018-09-28 07:12:53 UTC",
        "WatchdogTimestampMonotonic": "600204696",
        "WatchdogUSec": "3min"
    }
}
META: ran handlers
META: ran handlers
 [WARNING]: Unable to parse /etc/ansible/hosts as an inventory source

 [WARNING]: No inventory was parsed, only implicit localhost is available

 [WARNING]: provided hosts list is empty, only localhost is available. Note
that the implicit localhost does not match 'all'

localhost | SUCCESS => {
    "changed": true,
    "enabled": false,
    "name": "systemd-journald",
    "state": "stopped",
    "status": {
        "ActiveEnterTimestamp": "Fri 2018-09-28 07:13:18 UTC",
        "ActiveEnterTimestampMonotonic": "624728055",
        "ActiveExitTimestamp": "Fri 2018-09-28 07:13:18 UTC",
        "ActiveExitTimestampMonotonic": "624709766",
        "ActiveState": "active",
        "After": "systemd-journald.socket systemd-journald-audit.socket syslog.socket systemd-journald-dev-log.socket system.slice",
        "AllowIsolate": "no",
        "AmbientCapabilities": "",
        "AssertResult": "yes",
        "AssertTimestamp": "Fri 2018-09-28 07:13:18 UTC",
        "AssertTimestampMonotonic": "624713024",
        "Before": "sysinit.target",
        "BlockIOAccounting": "no",
        "BlockIOWeight": "[not set]",
        "CPUAccounting": "no",
        "CPUQuotaPerSecUSec": "infinity",
        "CPUSchedulingPolicy": "0",
        "CPUSchedulingPriority": "0",
        "CPUSchedulingResetOnFork": "no",
        "CPUShares": "[not set]",
        "CPUUsageNSec": "[not set]",
        "CPUWeight": "[not set]",
        "CacheDirectoryMode": "0755",
        "CanIsolate": "no",
        "CanReload": "no",
        "CanStart": "yes",
        "CanStop": "yes",
        "CapabilityBoundingSet": "cap_chown cap_dac_override cap_dac_read_search cap_fowner cap_setgid cap_setuid cap_sys_ptrace cap_sys_admin cap_audit_control cap_mac_override cap_syslog",
        "CollectMode": "inactive",
        "ConditionResult": "yes",
        "ConditionTimestamp": "Fri 2018-09-28 07:13:18 UTC",
        "ConditionTimestampMonotonic": "624713024",
        "ConfigurationDirectoryMode": "0755",
        "ControlGroup": "/system.slice/docker.service/system.slice/systemd-journald.service",
        "ControlPID": "0",
        "DefaultDependencies": "no",
        "Delegate": "no",
        "Description": "Journal Service",
        "DevicePolicy": "auto",
        "Documentation": "man:systemd-journald.service(8) man:journald.conf(5)",
        "DynamicUser": "no",
        "ExecMainCode": "0",
        "ExecMainExitTimestampMonotonic": "0",
        "ExecMainPID": "4116",
        "ExecMainStartTimestamp": "Fri 2018-09-28 07:13:18 UTC",
        "ExecMainStartTimestampMonotonic": "624713998",
        "ExecMainStatus": "0",
        "ExecStart": "{ path=/lib/systemd/systemd-journald ; argv[]=/lib/systemd/systemd-journald ; ignore_errors=no ; start_time=[Fri 2018-09-28 07:13:18 UTC] ; stop_time=[n/a] ; pid=4116 ; code=(null) ; status=0/0 }",
        "FailureAction": "none",
        "FileDescriptorStoreMax": "4224",
        "FragmentPath": "/lib/systemd/system/systemd-journald.service",
        "GID": "[not set]",
        "GuessMainPID": "yes",
        "IOAccounting": "no",
        "IOSchedulingClass": "0",
        "IOSchedulingPriority": "0",
        "IOWeight": "[not set]",
        "IPAccounting": "no",
        "IPEgressBytes": "18446744073709551615",
        "IPEgressPackets": "18446744073709551615",
        "IPIngressBytes": "18446744073709551615",
        "IPIngressPackets": "18446744073709551615",
        "Id": "systemd-journald.service",
        "IgnoreOnIsolate": "no",
        "IgnoreSIGPIPE": "yes",
        "InactiveEnterTimestamp": "Fri 2018-09-28 07:13:18 UTC",
        "InactiveEnterTimestampMonotonic": "624711670",
        "InactiveExitTimestamp": "Fri 2018-09-28 07:13:18 UTC",
        "InactiveExitTimestampMonotonic": "624714074",
        "InvocationID": "6e1c5db887b24abb8ecf1f1ab2871387",
        "JobRunningTimeoutUSec": "infinity",
        "JobTimeoutAction": "none",
        "JobTimeoutUSec": "infinity",
        "KeyringMode": "private",
        "KillMode": "control-group",
        "KillSignal": "15",
        "LimitAS": "infinity",
        "LimitASSoft": "infinity",
        "LimitCORE": "infinity",
        "LimitCORESoft": "infinity",
        "LimitCPU": "infinity",
        "LimitCPUSoft": "infinity",
        "LimitDATA": "infinity",
        "LimitDATASoft": "infinity",
        "LimitFSIZE": "infinity",
        "LimitFSIZESoft": "infinity",
        "LimitLOCKS": "infinity",
        "LimitLOCKSSoft": "infinity",
        "LimitMEMLOCK": "16777216",
        "LimitMEMLOCKSoft": "16777216",
        "LimitMSGQUEUE": "819200",
        "LimitMSGQUEUESoft": "819200",
        "LimitNICE": "0",
        "LimitNICESoft": "0",
        "LimitNOFILE": "16384",
        "LimitNOFILESoft": "16384",
        "LimitNPROC": "infinity",
        "LimitNPROCSoft": "infinity",
        "LimitRSS": "infinity",
        "LimitRSSSoft": "infinity",
        "LimitRTPRIO": "0",
        "LimitRTPRIOSoft": "0",
        "LimitRTTIME": "infinity",
        "LimitRTTIMESoft": "infinity",
        "LimitSIGPENDING": "59668",
        "LimitSIGPENDINGSoft": "59668",
        "LimitSTACK": "infinity",
        "LimitSTACKSoft": "8388608",
        "LoadState": "loaded",
        "LockPersonality": "yes",
        "LogLevelMax": "-1",
        "LogsDirectoryMode": "0755",
        "MainPID": "4116",
        "MemoryAccounting": "no",
        "MemoryCurrent": "[not set]",
        "MemoryDenyWriteExecute": "yes",
        "MemoryHigh": "infinity",
        "MemoryLimit": "infinity",
        "MemoryLow": "0",
        "MemoryMax": "infinity",
        "MemorySwapMax": "infinity",
        "MountAPIVFS": "no",
        "MountFlags": "",
        "NFileDescriptorStore": "0",
        "NRestarts": "0",
        "Names": "systemd-journald.service",
        "NeedDaemonReload": "no",
        "Nice": "-1",
        "NoNewPrivileges": "no",
        "NonBlocking": "no",
        "NotifyAccess": "main",
        "OOMScoreAdjust": "0",
        "OnFailureJobMode": "replace",
        "PermissionsStartOnly": "no",
        "Perpetual": "no",
        "PrivateDevices": "no",
        "PrivateNetwork": "no",
        "PrivateTmp": "no",
        "PrivateUsers": "no",
        "ProtectControlGroups": "no",
        "ProtectHome": "no",
        "ProtectKernelModules": "no",
        "ProtectKernelTunables": "no",
        "ProtectSystem": "no",
        "RefuseManualStart": "no",
        "RefuseManualStop": "no",
        "RemainAfterExit": "no",
        "RemoveIPC": "no",
        "Requires": "systemd-journald.socket system.slice",
        "Restart": "always",
        "RestartUSec": "0",
        "RestrictNamespaces": "yes",
        "RestrictRealtime": "yes",
        "Result": "success",
        "RootDirectoryStartOnly": "no",
        "RuntimeDirectoryMode": "0755",
        "RuntimeDirectoryPreserve": "no",
        "RuntimeMaxUSec": "infinity",
        "SameProcessGroup": "no",
        "SecureBits": "0",
        "SendSIGHUP": "no",
        "SendSIGKILL": "yes",
        "Slice": "system.slice",
        "StandardError": "inherit",
        "StandardInput": "null",
        "StandardInputData": "",
        "StandardOutput": "null",
        "StartLimitAction": "none",
        "StartLimitBurst": "5",
        "StartLimitIntervalUSec": "10s",
        "StartupBlockIOWeight": "[not set]",
        "StartupCPUShares": "[not set]",
        "StartupCPUWeight": "[not set]",
        "StartupIOWeight": "[not set]",
        "StateChangeTimestamp": "Fri 2018-09-28 07:13:18 UTC",
        "StateChangeTimestampMonotonic": "624728055",
        "StateDirectoryMode": "0755",
        "StatusErrno": "0",
        "StatusText": "Processing requests...",
        "StopWhenUnneeded": "no",
        "SubState": "running",
        "SuccessAction": "none",
        "SyslogFacility": "3",
        "SyslogLevel": "6",
        "SyslogLevelPrefix": "yes",
        "SyslogPriority": "30",
        "SystemCallArchitectures": "native",
        "SystemCallErrorNumber": "0",
        "SystemCallFilter": "~_sysctl add_key adjtimex afs_syscall bdflush break chroot clock_adjtime clock_settime create_module delete_module finit_module ftime get_kernel_syms getpmsg gtty idle init_module ioperm iopl kexec_file_load kexec_load keyctl lock lookup_dcookie modify_ldt mount mpx pciconfig_iobase pciconfig_read pciconfig_write perf_event_open pivot_root process_vm_readv process_vm_writev prof profil ptrace putpmsg query_module reboot request_key rtas s390_pci_mmio_read s390_pci_mmio_write s390_runtime_instr security settimeofday sgetmask spu_run ssetmask stime stty swapoff swapon switch_endian sysfs tuxcall ulimit umount umount2 uselib ustat vm86 vm86old vserver",
        "TTYReset": "no",
        "TTYVHangup": "no",
        "TTYVTDisallocate": "no",
        "TasksAccounting": "yes",
        "TasksCurrent": "3",
        "TasksMax": "4915",
        "TimeoutStartUSec": "1min 30s",
        "TimeoutStopUSec": "1min 30s",
        "TimerSlackNSec": "50000",
        "Transient": "no",
        "TriggeredBy": "systemd-journald.socket systemd-journald-audit.socket systemd-journald-dev-log.socket",
        "Type": "notify",
        "UID": "[not set]",
        "UMask": "0022",
        "UnitFilePreset": "enabled",
        "UnitFileState": "static",
        "UtmpMode": "init",
        "WantedBy": "sysinit.target",
        "Wants": "systemd-journald.socket systemd-journald-audit.socket systemd-journald-dev-log.socket",
        "WatchdogTimestamp": "Fri 2018-09-28 07:13:18 UTC",
        "WatchdogTimestampMonotonic": "624728426",
        "WatchdogUSec": "3min"
    }
}
```

### PR DESCRIPTION
**Pull request by fconil**
##### SUMMARY
Fixes #46247

Fixes both service and systemd modules

##### ISSUE TYPE

- Bugfix Pull Request

##### COMPONENT NAME

service
systemd

##### ANSIBLE VERSION

```paste below
2.8
```

##### ADDITIONAL INFORMATION

```paste below

```

### CODE REVIEW

**Review by maxamillion — APPROVED** [review]

**Comment by maxamillion** [comment]
shipit

**Comment by pilou-** [comment]
shipit

**Comment by maxamillion** [comment]
I think @bcoca had some concerns with this patch but I can't remember what they were.

**Comment by pilou-** [comment]
Here it is. bcoca ca wrote on #ansible-devel:
> i thought we already had this discussion, its doing a negative match, it should specifically look for a return and not assume that if we failed to enable is cause service is not possible to enable

**Review by gundalow — CHANGES_REQUESTED** [review]
Needs update based on Pilou's /bcoca's feedback above

**Comment by maxamillion** [comment]
waiting_on_contributor

File: lib/ansible/modules/system/systemd.py [hunk 1]
```diff
@@ -440,7 +440,10 @@ def main():
 
             # check systemctl result or if it is a init script
             if rc == 0:
-                enabled = True
+                enabled = out.startswith('enable')
+                if not enabled:
```
Comment (pilou-) [inline lib/ansible/modules/system/systemd.py:444]: Should not `action` value be tested here ?

**Comment by pilou-** [comment]
> Here it is. bcoca ca wrote on #ansible-devel:
>> i thought we already had this discussion, its doing a negative match, it should specifically look for a return and not assume that if we failed to enable is cause service is not possible to enable

@bcoca: which module is concerned (`service` or `systemd`) ? which line ?

**Comment by jborean93** [comment]
Due to the age of the PR and the concerns/feedback raised on the PR I'm going to close this.

**Pull request closed without merging**